src/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Dosis:wght@400;700;800&family=Open+Sans:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Weather</title>
+  </head>
+  <body>
+    <header class="fold">
+      <div class="form-container">
+        <form class="search-frm invisible">
+          <input
+            class="search-inp"
+            type="text"
+            placeholder="Enter city name..."
+          />
+          <button type="submit" class="search-btn">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
+              stroke="currentColor"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              class="feather feather-arrow-right-circle search-btn-icon"
+            >
+              <circle cx="12" cy="12" r="10"></circle>
+              <polyline points="12 16 16 12 12 8"></polyline>
+              <line x1="8" y1="12" x2="16" y2="12"></line>
+            </svg>
+          </button>
+        </form>
+        <div class="units">
+          <div class="input-group">
+            <input type="radio" name="degScale" id="degCel" checked="checked" />
+            <label for="degCel">ºC</label>
+          </div>
+          <div class="input-group">
+            <input type="radio" name="degScale" id="degFah" />
+            <label for="degFah">ºF</label>
+          </div>
```

Comment:
According to the project requirement, you're supposed to use `toggle` features not `radio button` features.
![image](https://user-images.githubusercontent.com/31897434/85640096-c24c3780-b682-11ea-86bc-5e893a0a69c4.png)

NOTE: toggling means you can change the display by clicking a button, In another, you should be able to switch from C to F without using the form.

Reply:
I left this option inside the form because of this requirement in the project specific section:
- Please make different requests for the Celsius data and the Fahrenheit data. 

Each option produces a different parameter in the API call, hence making different requests.

Reply:
Corrected